src/go/rpk/pkg/cli/cmd/redpanda/package.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package redpanda
```

Comment:
What's the purpose of this file?

Reply:
It's in the commit's description, but I will add it to the file itself:
> Create an "empty" file (src/go/rpk/pkg/cli/cmd/redpanda/package.go to prevent darwin builds from failing, since all the other files in the "cmd/redpanda" package are excluded.

Reply:
Done.

Reply:
Ah thanks -- I reviewed the pr in total, not the individual commits. Thanks!